src/domain/landingPage/landingPageHero/__tests__/LandingPageHero.test.tsx

```diff
@@ -1,13 +1,76 @@
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import * as React from 'react';
 
-import mockLandingPage from '../../__mocks__/landingPage';
-import LandingPageHero from '../LandingPageHero';
+import { LandingPageFieldsFragment } from '../../../../generated/graphql';
+import { fakeLandingPage } from '../../../../util/mockDataUtils';
+import LandingPageHero, { testIds } from '../LandingPageHero';
 
-test('matches snapshot', () => {
-  const { container } = render(
-    <LandingPageHero landingPage={mockLandingPage} />
+const title = 'Landing page title';
+const description = 'Landing page description';
+const buttonText = 'Button text';
+const landingPage = fakeLandingPage({
+  buttonText: { fi: buttonText },
+  description: { fi: description },
+  title: { fi: title },
+}) as LandingPageFieldsFragment;
+
+test('should be rendered correctly', () => {
+  render(<LandingPageHero landingPage={landingPage} />);
+
+  expect(screen.getByText(title)).toBeInTheDocument();
+  expect(screen.getByText(description)).toBeInTheDocument();
+  expect(screen.getByText(buttonText)).toBeInTheDocument();
+});
+
+test('should set text wrapper max-width', () => {
+  render(<LandingPageHero landingPage={landingPage} />);
+
+  expect(
+    (screen.getByTestId(testIds.content) as HTMLDivElement).style.maxWidth
+  ).toBe('520px');
+  expect(
+    (screen.getByTestId(testIds.content) as HTMLDivElement).style
+      .backgroundColor
+  ).toBe('rgba(255, 255, 255, 0.7)');
+});
+
+test('should set text wrapper max-width', () => {
+  global.innerWidth = 1000;
+  render(
+    <LandingPageHero
+      landingPage={{
+        ...landingPage,
+        titleAndDescriptionColor: { fi: 'WHITE' },
+      }}
+    />
   );
 
-  expect(container.firstChild).toMatchSnapshot();
+  expect(
+    (screen.getByTestId(testIds.content) as HTMLDivElement).style.maxWidth
+  ).toBe('340px');
+});
+
+test('should set text wrapper background color', () => {
+  render(
+    <LandingPageHero
+      landingPage={{
+        ...landingPage,
+        titleAndDescriptionColor: { fi: 'WHITE' },
+      }}
+    />
+  );
+
+  expect(
+    (screen.getByTestId(testIds.content) as HTMLDivElement).style
```

Comment:
.toHaveStyle could be used to make there little bit nicer

Reply:
Changed as suggested